hash/ibm5170_cdrom.xml

```diff
@@ -6983,6 +7012,59 @@ Incredibly slow in tearing down even on pcipcs7 (bypass with fast forward)
 		</part>
 	</software>
 
+	<software name="corellindx" supported="partial">
+		<description>Corel Linux Deluxe</description>
+		<year>1999</year>
+		<publisher>Corel</publisher>
+		<notes><![CDATA[
+RTM date: November 15, 1999
+Linux Kernel: 2.2.12 + Base Packages
+CHDs generated from retail CDs
+Installation and Open Circulation CDs are bootable.
+]]></notes>
+		<part name="cdrom1" interface="cdrom">
+			<feature name="part_id" value="Installation CD-ROM"/>
+			<feature name="part_number" value="JB#44675"/>
+			<diskarea name="cdrom">
+				<disk name="corel_linux_os_deluxe_install" sha1="dc70d14ee57d2c78a419d7422de8f9ffd498a4a9" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<feature name="part_id" value="Application CD-ROM"/>
+			<feature name="part_number" value="JB#44675"/>
+			<diskarea name="cdrom">
+				<disk name="corel_linux_os_deluxe_applications" sha1="89e7862346f3a0db5b877c4732b6311cad50f11c" />
+			</diskarea>
+		</part>
+		<part name="cdrom3" interface="cdrom">
+			<feature name="part_id" value="Open Circulation CD-ROM"/>
+			<feature name="part_number" value="JB#44150"/>
+			<diskarea name="cdrom">
+				<disk name="corel_linux_os_deluxe_open_circulation" sha1="99d02e94329115568afef4cc2c9e9583a34729f5" />
+			</diskarea>
+		</part>
+		<part name="cdrom4" interface="cdrom">
+			<feature name="part_id" value="Open Circulation CD-ROM (Alt)"/>
+			<feature name="part_number" value="JB#44150"/>
+			<diskarea name="cdrom">
+				<disk name="corel_linux_os_deluxe_open_circulation_alt" sha1="e66def04fb73155594a946cbee131aea35b79767" />
+			</diskarea>
+		</part>
+		<part name="cdrom5" interface="cdrom">
+			<feature name="part_id" value="Source Code CD-ROM"/>
+			<feature name="part_number" value="JB#44150"/>
+			<diskarea name="cdrom">
+				<disk name="corel_linux_os_deluxe_source_code" sha1="f435e52979b833c3ac00aaa05ddfdaf41bee5d8a"/>
+			</diskarea>
+		</part>
+		<part name="flop1" interface="floppy_3_5">
+		<!-- Package included a boot floppy for non El-Torito systems, but could not be dumped. This image was extracted from the Installation CD-ROM, located in BOOT\BOOT1440.IMG -->
+			<dataarea name="flop" size="1474560">
+				<rom name="corel_linux_boot_floppy.img" size="1474560" crc="4ac616ea" sha1="2f7999a3fd24b59b6939b94af47cf6cc599f3da0" status="baddump" />
+			</dataarea>
+		</part>
```

Comment:
FWIW that’s the “virtual floppy” image used for El Torito floppy emulation mode.  There’s no guarantee it’s the same as the real supplied floppy.

Reply:
It actually *is* a custom version, with a Corel Linux bootup screen. I agree that it might not match the floppy 1:1, but it's probably close to the real thing.